app/src/main/java/org/mozilla/fenix/settings/about/AboutFragment.kt

```diff
@@ -168,13 +168,11 @@ class AboutFragment : Fragment(), AboutPageListener {
     }
 
     private fun openLibrariesPage() {
-        startActivity(Intent(context, OssLicensesMenuActivity::class.java))
-        OssLicensesMenuActivity.setActivityTitle(
-            getString(
-                R.string.open_source_licenses_title,
-                appName
-            )
-        )
+        val ctx = context
+        if (ctx != null) {
```

Comment:
This function shouldn't get called when context is null, so `requireContext()` should be fine.

Reply:
True. This is done :ballot_box_with_check: